### PR TITLE
Update coworking-collabcafe.md to fix issue #2268

### LINKED ISSUE #2268: [BUG] Link in Online Collaboration Cafe section not working
### Description

The **GitHub page** link in the [**Online Collaboration Cafe**](https://the-turing-way.netlify.app/community-handbook/coworking/coworking-collabcafe.html#) section (see screenshot below) sends me to nowhere (`page not found`).

image

TODOs:

- [ ] Find out what the correct link should be.
- [ ] Replace link.

I think this is a perfect beginner issue.

#### Expected behaviour

Link should send me to GitHub page mentioned.

#### Actual behaviour

Page showing `Page not found`

image

### PR DESCRIPTION
**Pull request by eddybruv**
### Summary

Fixes #2268 

### List of changes proposed in this PR (pull-request)

* Found and replaced a broken link

### What should a reviewer concentrate their feedback on?

TODOs:

- [x] Find out what the correct link should be.
- [x] Replace link.

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file: @HeidiSeibold

### CODE REVIEW

**Comment by HeidiSeibold** [comment]
Thanks @eddybruv for this! 👏 

Currently you are linking to a file that is in the archive (I think that means it is not updated any longer).
https://github.com/alan-turing-institute/the-turing-way/blob/main/project_management/archive/online-collaboration-cafe.md

--- 
Screenshot of the file showing only dates from 2020 or earlier:

image
---

Can someone maybe help figuring out if there is a better file to link to or if we need to rewrite this section altogether? 
I saw @malvikasharan moved the file to archive. Can you help us? I am a bit confused as to what the right solution could be.

File: book/website/community-handbook/coworking/coworking-collabcafe.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Online Collaboration Cafe
 
 _The Turing Way_ Collaboration Cafes (or 'Cafés' with an accent) are virtual co-working spaces that are held fortnightly on the first and third Wednesdays of each month for 2 hours.
-The schedule will be updated periodically on the [GitHub page](https://github.com/alan-turing-institute/the-turing-way/blob/main/project_management/online-collaboration-cafe.md) as per the community requirement.
+The schedule will be updated periodically on the [GitHub page](https://github.com/alan-turing-institute/the-turing-way/blob/main/project_management/archive/online-collaboration-cafe.md) as per the community requirement.
```
Comment (malvikasharan) [inline book/website/community-handbook/coworking/coworking-collabcafe.md:5]:
```suggestion
The schedule will be updated periodically on our [Community Calendar](https://calendar.google.com/calendar?cid=dGhldHVyaW5nd2F5QGdtYWlsLmNvbQ) as per the community requirement.
```

**Review by malvikasharan — APPROVED** [review]
Thank you for raising this error. I have replaced the link with Calendar link that is always up to date. 🙌

**Comment by malvikasharan** [comment]
@all-contributors please add @eddybruv for bug

**Pull request merged**